dev-db/gigabase/gigabase-3.83-r2.ebuild

```diff
@@ -48,15 +54,6 @@ src_test() {
 	[[ $failcnt != 0 ]] && die "See warnings above for tests that failed"
 }
 
-src_install() {
```

Comment:
are you sure you can replace all this with the default? What about deleting `.la` files (these need to be deleted as a policy set by Gentoo)? Check with the previous version that the installed files are the same, and delete .la files not with `prune_libtool_files`, but rather using the code from https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Handling_Libtool_Archives